packages/logger/src/winston.ts

```diff
@@ -46,7 +48,7 @@ export class WinstonLogger implements Logger {
   static createWinstonInstance(options: Partial<LoggerOptions> = {}, transports?: winston.transport[]): Winston {
     const defaultMeta: DefaultMeta = {module: options?.module || ""};
 
-    return winston.createLogger({
+    return createLogger({
```

Comment:
why remove it here? this contradicts what is stated in the comment above
> We want to keep `winston` export as it's more readable and easier to understand

Reply:
The only function used in this file from `winston` is destructured. The namespace left for just the types.

Reply:
> The only function used in this file from winston is destructured.

My question was rather why as `winston.createLogger` is clearer to me than `createLogger` (without looking at imports)